docs/manifest.json

```diff
@@ -427,7 +427,7 @@
           }
         },
         {
-          "title": "next/future/image (experimental)",
+          "title": "next/future/image",
```

Comment:
Do we need to add a redirect in here?

Reply:
No, the url will remain the same: https://nextjs.org/docs/api-reference/next/future/image